setup.cfg

```
@@ -55,8 +55,8 @@ packages = find_namespace:
 python_requires = >=3.7
 include_package_data = True
 install_requires =
-    metomi-rose==2.0rc1
-    cylc-flow==8.0rc1.*  # the .* permits dev versions
+    metomi-rose==2.0rc2
```

I've shifted metomi-rose over to the `.dev` convention (especially as it was what caused me to jigger up the 1.0.0 release)

```suggestion
    metomi-rose==2.0rc2.*  # the .* permits dev versions
```